websites/C/Cuevana 3/presence.ts

```diff
@@ -30,21 +30,32 @@ presence.on("UpdateData", async () => {
     document.location.pathname.includes("/episodio")
   ) {
     const timestamps = presence.getTimestamps(
-        Math.floor(video.currentTime),
-        Math.floor(video.duration)
+      Math.floor(video.currentTime),
+      Math.floor(video.duration)
       ),
```

Comment:
just use presence.getVideoTimestamps or how its called

Reply:
K Let me check

Reply:
these are the only functions I found
![image](https://user-images.githubusercontent.com/27692047/109394782-59dae780-78ee-11eb-864d-e07e9de5c827.png)

Reply:
ok i think i got it

Reply:
presence.getTimestampsfromMedia()

Reply:
I don't think I can use it since the iframe.ts is doing all the heavy lifting of the video, thats why I'm using presence.getTimestamps() instead. 🤔

Reply:
I don't think I can use it since the iframe.ts is doing all the heavy lifting of the video (the video is within an iframe), thats why I'm using presence.getTimestamps() instead. 🤔

Reply:
tried to do something like this:
![image](https://user-images.githubusercontent.com/27692047/109401520-e13b5180-7914-11eb-931b-a7542a3b663e.png)
but ending up getting undefined 😢:
![image](https://user-images.githubusercontent.com/27692047/109401604-3a0aea00-7915-11eb-8e32-da370d3effb6.png)